src/components/cursor.js

```diff
@@ -319,6 +319,7 @@ module.exports.Component = registerComponent('cursor', {
   },
 
   onEnterVR: function () {
+    if (this.data.rayOrigin === 'mouse') { this.clearCurrentIntersection(true); }
```

Comment:
Can we just call `this.clearCurrentIntersection(true);`. Do we need to check for `mouse`?

Reply:
The point is correct. I removed the mouse check.